rest/views.py

```diff
@@ -137,7 +138,7 @@ def get_queryset(self):
             slug=self.kwargs['vocab_slug']
         )
         if not vocabs.exists():
-            return []
+            raise Http404()
```

Comment:
This seems like it might be out of scope?  Changing a return to a raise seems like a pretty big API change to add in with cascaded deletes.